### PR TITLE
[bugfix][RHEL/5] fix `make validate` failures for RHEL/5

### PR DESCRIPTION
**Pull request by redhatrises**
- Removed unreferenced OVAL content
- Add `make validate` check to Makefile

### CODE REVIEW

File: RHEL/5/input/system/accounts/restrictions/account_expiration.xml [hunk 1]
```diff
@@ -95,7 +95,7 @@ If there are no duplicate IDs, no line will be returned.
 Unique IDs allow for accountability on the system. 
 </rationale>
 <ident stig="GEN000320" />
-<oval id="accounts_unique_uid"/>
+<!--oval id="accounts_unique_uid"/-->
```
Comment (shawndwells) [inline RHEL/5/input/system/accounts/restrictions/account_expiration.xml]: Does this map to the (now deleted) `````accounts_unique_id````` OVAL?

**Comment by redhatrises** [comment]
@shawndwells nice catch! Updated the PR with enabling accounts_unique_uid.xml

**Comment by redhatrises** [comment]
Checked the build failure, and it looks unrelated to this PR as the failure lies in `RHEL/6` specifically with `make validate`

**Comment by shawndwells** [comment]
content validates:

`````[shawnw@localhost 5]$ make validate
oscap xccdf validate-xml output/ssg-rhel5-xccdf.xml
oscap oval validate-xml --schematron output/ssg-rhel5-oval.xml
oscap cpe validate-xml output/ssg-rhel5-cpe-dictionary.xml
oscap oval validate-xml --schematron output/ssg-rhel5-cpe-oval.xml
oscap ds sds-validate output/ssg-rhel5-ds.xml
oscap xccdf validate-xml output/ssg-centos5-xccdf.xml
oscap ds sds-validate output/ssg-centos5-ds.xml
oscap xccdf validate-xml output/ssg-sl5-xccdf.xml
oscap ds sds-validate output/ssg-sl5-ds.xml
cd output; ../utils/verify-references.py --rules-with-invalid-checks --ovaldefs-unused ssg-rhel5-xccdf.xml`````

**Comment by shawndwells** [comment]
scan runs as expected:

`````
[shawnw@localhost 5]$ oscap xccdf eval --profile stig-rhel5-server-upstream output/ssg-rhel5-xccdf.xml 
Title   Ensure /tmp Located On Separate Partition
Rule    partition_for_tmp
Ident   GEN003624
Result  fail

Title   Ensure /var Located On Separate Partition
Rule    partition_for_var
Ident   GEN003621
Result  fail

Title   Ensure /var/log/audit Located On Separate Partition
Rule    partition_for_var_log_audit
Ident   GEN003623
Result  fail

Title   Ensure /home Located On Separate Partition
Rule    partition_for_home
Ident   GEN003620
Result  fail
......
`````

**Comment by shawndwells** [comment]
Thanks for the cleanup, @redhatrises !

**Pull request merged**